pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3388,95 @@ def set_view(self, viewType):
         if res == 0:
             raise RuntimeError('Failed to set view in Calendar')
 
+    # ----------------------------------------------------------------
+    def set_id(self, ID):
+        """
+        Set the calendar type
+        Receive only one parameter, which takes variants below:
+        'gregorian', 'gregorian_US', 'japan', 'taiwan', 'korea',
+        'hijri', 'thai', 'hebwer', 'gregorian_french_me',
+        'gregorian_arabic', 'gregorian_english_xlit',
+        'gregorian_french_xlit', 'umalqura'
+        """
+
+        dict_types = {
+            'gregorian': 1,
+            'gregorian_US': 2,
+            'japan': 3,
+            'taiwan': 4,

```

Comment:
CAL_ID integer constants must be defined in `win32defines.py` and used here instead of magic numbers.